pages/index.tsx

```diff
@@ -30,8 +33,14 @@ const Index: React.FC = () => {
           </Button>
         </div>
         <div className="grid grid-cols-2 items-center gap-2 mt-5">
-          <Button>Or Join with Google</Button>
-          <Button>Or Join with GitHub</Button>
+          <Button
+            onClick={async () =>
+            { await supabase.auth.signIn({ provider: 'google' }) }}
```

Comment:
Is there a reason for using async await in your onClick callback?
wont the browser execute the promise anyways?

Reply:
Correct,  `await` makes sense for `signout`, not sign in, I will remove this